### PR TITLE
add net commands

### PR DESCRIPTION
**Pull request by LexLuthr**
This PR implements `lotus net` commands for `boostd` and enables making changes to rcmanager config

### CODE REVIEW

**Review by dirkmc — APPROVED** [review]

File: cmd/boostd/net.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"github.com/filecoin-project/boost/node"
+	lcli "github.com/filecoin-project/lotus/cli"
+	"github.com/urfave/cli/v2"
+)
+
+var NetCmd = &cli.Command{
```
Comment (nonsense) [inline cmd/boostd/net.go:9]:
```suggestion
var netCmd = &cli.Command{
```

File: cmd/boostd/main.go [hunk 2]
```diff
@@ -47,6 +47,7 @@ func main() {
 			logCmd,
 			dagstoreCmd,
 			piecesCmd,
+			NetCmd,
```
Comment (nonsense) [inline cmd/boostd/main.go:50]:
```suggestion
			netCmd,
```

**Comment by LexLuthr** [comment]
@dirkmc Are we okay to merge with failed test?

**Comment by dirkmc** [comment]
Looks like it was just a flaky test, merging

**Pull request merged**